app/partials/actions/add.html

```diff
@@ -1,10 +1,10 @@
 <div class="container">
 	<h3>
-		Add {{rdesc.name.singular}}
+		Add {{rdesc.attributes.name.singular}}
 	</h3>
-	<h5>{{rdesc.description}}</h5>
+	<h5>{{rdesc.attributes.description}}</h5>
 
-	<div class="form-group" ng-repeat="field in rdesc.fields" ng-include="'/partials/fields/default-field.html'">
+	<div class="form-group" ng-repeat="field in rdesc.attributes.fields" ng-include="'/partials/fields/default-field.html'">
```

Comment:
@maxdumas Hmm... I'm not a fan of `constant`/`static` for fields that the user can never set, because there's nothing that says these fields' values will remain constant (and `modified`, for example, will change frequently). I do like `automatic`, though, since it's starting to get at the idea that these values are managed by an external agent. What do you think about `independent`?

For the "settable just once" properties, I actually think `final` is a great name. Any reason not to go with that?